### PR TITLE
Example of granting local users in wheel access to libvirt

### PR DESCRIPTION
**Pull request by eparis**

### CODE REVIEW

File: Documentation/dev/libvirt-howto.md [hunk 1]
```diff
@@ -43,7 +43,19 @@ qemu-img resize coreos_production_qemu_image.img +8G
 #### 1.4 Get a Tectonic License
 Go to https://account.coreos.com/ and obtain a Tectonic license. Save the *pull secret* and *license path* somewhere.
 
-#### 1.5 Prepare the configuration file
+#### 1.5 Make sure you have permisions for qemu:///system
```
Comment (wking) [inline Documentation/dev/libvirt-howto.md]: nit: I'd prefer backtics for `qemu:///system`, but that's obviously not a big deal either way ;).

**Comment by eparis** [comment]
tick'd

**Comment by wking** [comment]
Looks good to me, and tests are green.  Anyone with polkit experience want to chime in?  If not, I'll approve this in ~two hours.

**Comment by wking** [comment]
/lgtm

Although this is unlikely to actually land until we sort out the e2e-aws breakage (e.g. see [here][1] and later).

[1]: https://github.com/openshift/installer/pull/142#issuecomment-413641028

**Comment by wking** [comment]
> Although this is unlikely to actually land until we sort out the e2e-aws breakage...

Ah, saved by the retest-not-required-docs-only label :).

**Pull request merged**